src/client/layouts/default.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div @click="trackOutboundLink">
```

Comment:
Move to a component so it can be used in different places and you can use `this.$ga` instead of relying on `window.ga`?

Reply:
👍